skills/declarative/Calendar/Calendar/Program.cs

```diff
@@ -4,7 +4,7 @@
 using Microsoft.Extensions.Configuration;
 using Microsoft.Extensions.Hosting;
 
-namespace calendarskill
+namespace Calendar
```

Comment:
nit: just wondering if the namespace for our project should also include "Microsoft.Bot.*"